components/experimental/src/dimension/mod.rs

```diff
@@ -51,6 +51,13 @@ impl CurrencyFormatter {
         options: options::CurrencyFormatterOptions,
         error: DimensionError,
         #[cfg(skip)]
+        functions: [
```

Comment:
These are the default values for this parameter, so this PR doesn't change anything.

Reply:
Okay, got it